extensions/vscode/src/constants.ts

```diff
@@ -116,3 +116,9 @@ export const enum Views {
   HelpAndFeedback = "posit.publisher.helpAndFeedback",
   Logs = "posit.publisher.logs",
 }
+
+export const DebounceDelaysMS = {
+  file: 1000,
+  refreshRPackages: 1000,
+  refreshPythonPackages: 1000,
+};
```

Comment:
Debounce variables pulled out to constants, with slight differences per usages, so we can tune these.